cockpit/src/main/res/layout/cockpit_param_name.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is there any reason to wrap single `TextView` in `ConstraintLayout`?

Reply:
When you want to include a layout in a ConstraintLayout you have to wrap it with ConstraintLayout also.